### DISCUSSION TITLE
Switching from Experimental to Release target date

### DISCUSSION THREAD

**Discussion by rumax** [facebookexperimental/Recoil#1029] [General]
The project is still in "experimental" phase. Is there any decisions on target date to release production ready version? That might help for projects to decide when recoil can be used.

**Comment by MMRandy** [comment]
Agree 100% with @rumax here....how long will this project remain in Experimental mode?   Better yet...why would it be in that mode for so long...are there major usability issues to resolve still?  I love the simplicity of the lib and would love to use it...but don't want to throw myself under the bus for using an "experimental" lib.   Is there release date planned?  Thx...
